README.md

```diff
@@ -58,23 +64,8 @@ use `http:localhost` as the host for the WeCarry API, due to AzureAD's policies.
 (This affects the `AUTH_CALLBACK_URL` in the `.env` file and the `buffalo.environment.HOST` value
 in the docker-compose file.)
 
-### Facebook
-To add an organization using Facebook authentication, create a database organization record  
-that includes an auth_type of `facebook` and an auth_config like the following ... 
-
-```
-{}
-```
-
-The two environment variables `FACEBOOK_KEY` and `FACEBOOK_SECRET` 
-will need to be set for the appropriate Facebook oauth account and application.
-
-For local development, if you are using `http`, then you will likely need to 
-use `http:localhost` as the host for the WeCarry API, due to Facebook's policies.
-(This affects the `AUTH_CALLBACK_URL` in the `.env` file and the `buffalo.environment.HOST` value
-in the docker-compose file.)
+<h4 id="google"> Google </h4>
```

Comment:
Would it be better to leave this as markdown (####Google), since github will give it a link in the UI anyway.

Reply:
There are two headings "Google" so rather that add extra word(s) I just did it this way. Do you know of another way to do it?

Reply:
Maybe I change it to Google GSuite?

Reply:
Oh, ok. That should work.